src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: inherit;
+  font-size: inherit;
+  line-height: inherit;
+  color: inherit;
+}
```

Comment:
Use * selector only for box-sizing rule

![image](https://user-images.githubusercontent.com/80700247/225343112-6b284d07-e9f0-4ef7-9475-4ccffc57e2db.png)